source/interfaces.rst

```diff
@@ -89,7 +92,9 @@ non-volatile memory.
 
 On some CPUs the download can be done via USB. USB is convenient because it
 features a relatively high bandwidth and does not need expensive external
-components to be used.
+components to be used. Each board should present unique ID to the host. If IDs
+are identical for all boards there is no way to drive such devices in
+fully automated environment.
```

Comment:
I disagree. It is still possible to identify a device based on it's USB-path. I would prefer a _softer_ statement here.

Reply:
I tried and it didn't end well so I'll won't change my mind. But in principal you're right. I'll update wording.